docs/pages/api-docs/data-grid/grid-edit-row-api.json

```diff
@@ -5,7 +5,7 @@
     {
       "name": "commitCellChange",
       "description": "Updates the field at the given id with the value stored in the edit row model.",
-      "type": "(params: GridCommitCellChangeParams, event?: MuiBaseEvent) => boolean | Promise<boolean>"
+      "type": "(\n    params: GridCommitCellChangeParams,\n    event?: MuiBaseEvent,\n  ) => boolean | Promise<boolean>"
```

Comment:
I think we need to sanitize the output of `node.getText` to remove line breaks, spaces and the comma after the last value. A regex should do sufficient.

Reply:
Yep, I finished the grouping column and did not touch this PR today.
`getText` has a good output but I need to use it everywhere and to format correctly.

I don't know how it would impact perfs, but I would be great to use `prettier.format` to have a clean output.

A Regex should be good enough at 1st though, improving the formatting is not related to this PR.

Reply:
I used Prettier in the end, and the result is nice I think.